shopyo/config.py

```diff
@@ -29,7 +29,15 @@ class DevelopmentConfig(Config):
     ENV = "development"
     DEBUG = True
     # EXPLAIN_TEMPLATE_LOADING = True
-    LOGIN_DISABLED = True
+    LOGIN_DISABLED = False
+    # FLASK-MAILMAN CONFIGS
+    MAIL_SERVER = 'smtp.googlemail.com'
+    MAIL_PORT = 465
+    MAIL_USE_TLS = False
+    MAIL_USE_SSL = True
+    MAIL_USERNAME = os.environ.get("MAIL_USERNAME")
```

Comment:
since this is os.environ, mail_username is too generic as it will be confused amidst other machine settings,

maybe SHOPYO_MAIL_SERVER

else, if we gitignore config.py and config.josn, we can write it in here itself but will have to create the two files on init

Reply:
I thought the convention was to keep the same ENV variable name as defined in the configurations. In this case the config name MAIL_USERNAME name comes from flask-mailman. We can change it to SHOPYO_MAIL_USERNAME if that is a better approach.

Reply:
@rehmanis yes indeed. env vars are isolated inproduction, just on local machine unless maybe we use .env files

keep it as it is ....